src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -198,16 +201,16 @@ export class ExchangeWrapper extends ContractWrapper {
         );
         this._throwErrorLogsAsErrors(response.logs);
         const logFillArgs = response.logs[0].args as LogFillContractEventArgs;
-        const filledAmount = new BigNumber(logFillArgs.filledValueT);
-        return filledAmount;
+        const filledTakerTokenAmount = new BigNumber(logFillArgs.filledTakerTokenAmount);
+        return filledTakerTokenAmount;
     }
     /**
      * Sequentially and atomically fills signedOrders up to the specified takerTokenFillAmount.
      * If the fill amount is reached - it succeeds and does not fill the rest of the orders.
      * If fill amount is not reached - it fills as much of the fill amount as possible and succeeds.
      * @param   signedOrders            The array of signedOrders that you would like to fill until
      *                                  takerTokenFillAmount is reached.
-     * @param   takerTokenFillAmount    The total amount of the takerTokens you would like to fill.
+     * @param   fillTakerTokenAmount    The total amount of the takerTokens you would like to fill.
      * @param   shouldCheckTransfer     Whether or not you wish for the contract call to throw if upon
```

Comment:
Forgot to rename comment param to `shouldThrowOnInsufficientBalanceOrAllowance `.